primeqa/ir/dense/colbert_top/colbert/infra/config/settings.py

```diff
@@ -163,7 +163,7 @@ class TrainingSettings:
     # used in distillation (Student/Teacher) training
     student_teacher_temperature: float = DefaultVal(1.0)
     student_teacher_top_loss_weight: float = DefaultVal(0.5)
-    teacher_model_type: str = DefaultVal('xlm-roberta-base')
+    teacher_model_type: str = DefaultVal('bert')
```

Comment:
Can we call them `encoder` family as opposed to `bert` ? Are we tied to BERT in general?

Reply:
The code now supports models from three model families, `bert`, `roberta` and `xlm-roberta`.  Adding any encoder will be not feasible out of the box.

Reply:
We should be consistent with the default setting what ever it is. I suggested xlm-roberta because PrimeQA leaderboard win is from Dr Decr which is xlm-roberta. Whatever, we choose that default is to be used everywhere.

Reply:
Makes sense.